examples/inspection/plot_permutation_importance_multicollinear.py

```diff
@@ -60,8 +60,7 @@
 fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(12, 8))
 ax1.barh(tree_indices,
          clf.feature_importances_[tree_importance_sorted_idx], height=0.7)
-ax1.set_yticklabels(data.feature_names[tree_importance_sorted_idx])
-ax1.set_yticks(tree_indices)
+
```

Comment:
same as above

Reply:
Noted

Reply:
I meant you need the same code change here as in the file plot_permutation_importance.py, i.e. bring the y-tick-labels back to the left figure.

Reply:
Thank you @lorentzenchr I will make the changes

Reply:
Could you just permute the 2 lines (as in my previous comment) instead of importing `FixedLocator`.